readme.txt

```diff
@@ -215,9 +215,30 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 == Changelog ==
 
-= [4.10] TBD =
-
+= [5.0.0] TBD =
+
+* Feature - Month, Day and List views HTML, CSS and JavaScript were completely refactored.
```

Comment:
This is sort of accurate. What we did was add _new_ Month, Day, and List views with new HTML, CSS, and JavaScript.

```suggestion
* Feature - Created new Month, Day, and List views with all new HTML, CSS, and JavaScript.
```